src/AzureClient/EntryPoint/EntryPointGenerator.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
﻿// Copyright (c) Microsoft Corporation.
```

Reply:
Is this recommended? From what I can see, just about every source file in IQ# says "All rights reserved." Happy to go through and delete it everywhere if that's the best practice.

Reply:
This is the latest guidance, so far as I understand, and should be used for new code. That said, if I understand correctly, existing code does not need to be updated.